awscli/examples/networkflowmonitor/get-query-status-workload-insights-top-contributors.rst

```diff
@@ -0,0 +1,13 @@
+**To retrieve the status of the query**
+
+The following ``get-query-status-workload-insights-top-contributors`` example displays the current status of the query in the specified account. ::
+
+    aws networkflowmonitor get-query-status-workload-insights-top-contributors \
+        --scope-id e21cda79-30a0-4c12-9299-d8629d76d8cf \
+        --query-id f2a87c70-3e5a-362e-8beb-4747d13d8419
+
+Output::
+
+    {
+        "status": "SUCCEEDED"
+    }
```

Comment:
Every example needs a for more information link.
https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link